scripts/code_review_tool_evaluator.py

```diff
@@ -33,7 +33,7 @@
 from bugbug.vectordb import QdrantVectorDB
 
 code_review.TARGET_SOFTWARE = "Mozilla Firefox"
-VERBOSE_CODE_REVIEW = False
+VERBOSE_CODE_REVIEW = True
```

Comment:
Was this change for debugging or do you intend to set it to True by default?

Reply:
It was for debugging. I did undo it in 5e94ddceca596aac5d3dcb78e14313b11d62688b.